db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20171014192702) do
+ActiveRecord::Schema.define(version: 20171030162744) do
```

Comment:
The only thing that worries me is that this collation changes have no effect on the schema, so if new installations just do a `schema:load`, this collations won't be honored... only on systems that run this migration.

Reply:
Is that why Travis failed?